pkgs/development/python-modules/sphinxcontrib-fulltoc/default.nix

```
@@ -0,0 +1,26 @@
+{ lib, buildPythonPackage, fetchPypi, sphinx, pbr }:
+
+buildPythonPackage rec {
+  pname = "sphinxcontrib-fulltoc";
+  version = "1.2.0";
```

```suggestion
  version = "1.2.0";
  disabled = isPy27;
```
not sure about this one, pkgutil namespaces are broken in nixpkgs (because they can't scan multiple directories).

But python2 is EOL, so I think adding new packages which don't support it, is okay.

---

Fixed (but slightly differently). Thanks!